styles/components/infoTooltip/_tooltipComponent.scss

```diff
@@ -1,4 +1,4 @@
-@import "../global/index";
+@import "../../global/index";
 
 .tooltip-wrapper {
```

Comment:
Perhaps before this gets too big we should migrate all styles to the prefix you've used? Maybe in a follow up PR?